opentreemap/treemap/instance.py

```diff
@@ -449,6 +451,22 @@ def is_accessible_by(self, user):
         except ObjectDoesNotExist:
             return False
 
+    def export_enabled_for(self, user):
+        if self.feature_enabled('exports'):
+            if self.non_admins_can_export:
+                return True
+            else:
+                if user.is_authenticated():
+                    iuser = user.get_instance_user(self)
+                    return iuser and iuser.admin
```

Comment:
if `iuser` is falsy but not `False`, this will sometimes fail to return a `Boolean`. Since the rest of this function clearly returns `True` or `False`, consider making this line do it too, for consistency. I understand the callers will just be doing truthy/falsy checks so this is not an issue now, but it can't hurt in case somewhere down the line a different kind of check is added.